packages/material-ui-lab/src/TimelineContent/timelineContentClasses.js

```diff
@@ -0,0 +1,14 @@
+import { generateUtilityClass, generateUtilityClasses } from '@material-ui/unstyled';
```

Comment:
Please also add timlineContentClasses.d.ts and update the index.d.ts file

Reply:
Done!